ui/job-view/headerbars/PrimaryNavBar.jsx

```diff
@@ -11,69 +11,56 @@ import FiltersMenu from './FiltersMenu';
 import HelpMenu from './HelpMenu';
 import SecondaryNavBar from './SecondaryNavBar';
 
-export default class PrimaryNavBar extends React.Component {
-  constructor(props) {
-    super(props);
+export default function PrimaryNavBar(props) {
```

Comment:
Looks like a lot of changes, but really just passing ``filterModel`` to the right places and was able to simplify this down to a pure function.